Makefile

```diff
@@ -20,7 +20,7 @@ generate: codegen gqlgen protopb manifests yaml apidocs-gen
 
 docker-generate:
 	$(eval WORKDIR := /go/src/github.com/everoute/everoute)
-	docker run --rm -iu $$(id -u):$$(id -g) -w $(WORKDIR) -v $(CURDIR):$(WORKDIR) everoute/generate make generate
+	docker run --rm -iu 0:0 -w $(WORKDIR) -v $(CURDIR):$(WORKDIR) everoute/generate make generate
```

Comment:
This is weird. If user is not root, it would make `CURDIR` contain files with root permission, which is not expected.

Reply:
> contain files with root permission, which is not expected.

New generated file will have the same privilege with outside user, not the root permission

Files below were generated with `-iu 0:0`
```
-rw-r--r--  1 wcl  staff   4.3K Dec 17 11:03 agent.everoute.io_agentinfos.yaml
-rw-r--r--  1 wcl  staff   8.0K Dec 17 11:07 group.everoute.io_endpointgroups.yaml
-rw-r--r--  1 wcl  staff   2.7K Dec 17 11:03 group.everoute.io_groupmembers.yaml
-rw-r--r--  1 wcl  staff   5.4K Dec 17 11:03 group.everoute.io_groupmemberspatches.yaml
-rw-r--r--  1 wcl  staff   4.0K Dec 17 11:17 security.everoute.io_endpoints.yaml
-rw-r--r--  1 wcl  staff   3.3K Dec 17 11:03 security.everoute.io_globalpolicies.yaml
-rw-r--r--  1 wcl  staff    52K Dec 17 11:17 security.everoute.io_securitypolicies.yaml
```

Reply:
👌